linode/manager
# fix, style: [M3-6644] - Standardize styling of copy icon on OBJ bucket details page

## Description 📝
The original ask of this ticket was to fix the tooltip, which was being cut off on the left side at small screen sizes.

This ticket proposes a couple of other changes to the icon instead. The goal is to remove the single use of a custom implementation of a file copy icon and replace it with the copy icon we use throughout the rest of the app.

## Major Changes 🔄
- Swap out the custom styled file copy icon for a `CopyTooltip`component. 
   - We also get an  `aria-label` and tooltip text on hover (not just click).
- Move the placement of the tooltip to the right side of the breadcrumb, which is more consistent with how we display other copy icons. (e.g. right side of text fields for a linode's IP address and ssh access)
- UX approved the switch in icon type and placement, as well as requested the breadcrumb text size increase.

## Preview 📷

<details><summary>Videos 🎥 </summary>
The original issue:

https://github.com/linode/manager/assets/114685994/ab6ec6d5-2328-4496-a990-3b63cc9bdc05

The proposed fix:
| Before  | After   |
| ------- | ------- |
| ![ezgif com-video-to-gif](https://github.com/linode/manager/assets/114685994/ffd69289-242f-4f5b-a2d4-76ba67099389) | ![ezgif com-video-to-gif (1)](https://github.com/linode/manager/assets/114685994/316bb238-2977-4aa7-9cbb-6534ac1c8d4c) |
</details>

## How to test 🧪
1. **How to setup test environment?**
- Have OBJ enabled on an account and at least one bucket
 or
- Turn on the MSW and go to http://localhost:3000/object-storage/buckets/us-east-1/obj-bucket-0.
2. **How to reproduce the issue (if applicable)?**
- In prod, adjust the browser's screen size and note that the tooltip text is cut off on the left when the copy button is clicked on smaller screens.
3. **How to verify changes?**
- Verify that the tooltip text is not getting cut off on any screen size on this branch.
- Verify that the copy tooltip is still copying the file path to the clipboard on this branch.
